tox.ini

```diff
@@ -67,3 +67,14 @@ commands =
     mkdir -p docs/build
     sphinx-build -b html -Ean -j 2 -w build/sphinx-errors \
         docs/source docs/build
+
+[testenv:release]
+deps =
+    toml
+    pep517
+    flit
```

Comment:
You don't need flit and pep517 - pep517.build will download the build-system on the fly